force_wfmanager/central_pane/plot.py

```diff
@@ -195,3 +242,20 @@ def update_model(self):
             self.analysis_model.selected_step_index = None
         else:
             self.analysis_model.selected_step_index = selected_indices[0]
+
+    @on_trait_change('reset_plot')
+    def reset_pressed(self):
+        self.resize_plot()
+
+    def _get_reset_enabled(self):
+        x_data = self._plot_data.get_data('x')
+        if len(x_data) > 1:
+            return True
+        return False
+
+    def set_plot_range(self, x_low, x_high, y_low, y_high):
+        """Helper method to set the size of the current _plot"""
+        self._plot.range2d.x_range.low = x_low
```

Comment:
I suspect my comment is because of this. this clamps the zoom for some reason.

Reply:
So if you zoom out to the maximum and click reset, then try and zoom out within the newly reset plot it ... doesn't do anything. 
Perhaps there's some internal counter in chaco for how many 'zoom out' actions you're allowed which is not reset by resizing the plot?